static/src/stylesheets/module/_adslot.scss

```diff
@@ -139,10 +133,11 @@
         display: none;
     }
 
+    padding-left: $left-column-wide + $gs-gutter * 2;
```

Comment:
This allows the label and the advert to have the same margins, which will be responsive to the viewport size.